sdk/keyvault/keyvault-certificates/test/internal/challengeBasedAuthenticationPolicy.spec.ts

```diff
@@ -1,185 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
I think it's worthwhile to keep these tests - like unfortunately we can't run tests against KV Common itself so testing this at the package level is still necessary